bin/extract.sh

```diff
@@ -10,6 +10,19 @@ do
     echo "...  $name"
     cp $corpus txt/$name.txt
 done
+sed -i '/\[Illustration\]/d'  txt/*.txt
+sed -i '1,/START OF THE PROJECT GUTENBERG/d' txt/*.txt
+sed -i '/END OF THE PROJECT GUTENBERG/,$d' txt/*.txt
+sed -i '/^End of Project Gutenberg/d' txt/*.txt
+sed -i '/\*       \*/d' txt/*.txt 
+for f in txt/*-[1-9].txt
+do
+    volume=$(echo "$f" | sed 's/-[1-9].txt//g');
+    cat "$volume"-[1-9].txt > "$volume".txt
+done
+if [ -f "$volume".txt ] ;then
+    rm "$volume"-[1-9].txt
+fi
```

Comment:
Let’s put the `rm` within the `for` loop please.

This may work but the `$volume` variable has been defined **in** the `for` loop (= scope) and it can lead to unexpected results when used outside of its scope.